nengo/dists.py

```diff
@@ -156,12 +163,17 @@ class UniformHypersphere(Distribution):
 
     """
 
-    def __init__(self, surface=False):
-        self.surface = surface
+    def __init__(self, low=0, high=1, surface=False):
+        self.low = max(low, 0)
+        self.high = max(high, self.low)
+
+        if surface:
+            self.low = self.high
```

Comment:
This logic (of the interaction between high, low, and surface) should be documented in the parameters